### DISCUSSION TITLE
How to create postage_label's via the api?

### DISCUSSION THREAD

**Discussion by Braedencraig** [chitchats/chitchats-api-doc#30] [Q&A] [answered]
Hey there! 

Sorry if this is a dumb question, but i'm a little confused on how to go about creating `postage_labels` for specific shipments!
When I create a shipment, I see the fields, `postage_label_png_url`, and `postage_label_zpl_url` are null. Is there an extra step I am missing in order to create those labels properly?

**Comment by gshaw** [comment, marked as answer]
Labels are only available if the shipment has been purchased and in some cases not received.  If your shipment is purchased and not received by Chit Chats and you aren't seeing a label URL then reply with the shipment ID and if it's on staging or production and somebody can investigate as you should see a URL in that scenario.

**Reply by Braedencraig** [reply]
Thank you so much for the help!
